section2/exercises/if-statements.js

```diff
@@ -82,15 +91,21 @@ When cupsOfFlour = 2 and hasSauce = true, your program should log "I can make pi
 When cupsOfFlour = 3 and hasSauce = true, your program should log "I can make pizza";
 */
 
-var cupsOfFlour = 1;
+var cupsOfFlour = 0;
 var hasSauce = true;
 
 // Write your conditional here
+if (cupsOfFlour < 2){
```

Comment:
Revisit this exercise, too. You _should_ be using a logical operator to evaulate both `cupsOfFlour` AND `hasSauce`. Feels like maybe you were moving too quickly on this exercise?